lib/lhm/entangler.rb

```diff
@@ -11,6 +11,9 @@ class Entangler
 
     attr_reader :connection
 
+    MAX_RETRIES = 10
+    SLEEP_DURATION = 1
```

Comment:
These two values as defaults are fine.

Would it be possible to add a feature that will allow us to override them in some way, even using environment variables?

LHMs are used in multiple places in the Shopify organization, and in some cases we might need to be able to override these defaults.